### DISCUSSION TITLE
Setting up Deckboad

### DISCUSSION THREAD

**Discussion by Glycode** [negue/meme-box#372] [Q&A] [answered]
I'm currently trying to set up the deckboard extension. At the end the [setup](https://github.com/gigili/deckboard-memebox#setup) says to enter my connection information. When I enter the shown numbers nothing happens tho. Do I have to enter other specific number and if so where can I find them?

**Comment by Glycode** [comment, marked as answer]
Nvmd, solved it. I had to restart deckboard after entering the connection information. Also the port number was 4444

**Reply by gigili** [reply]
Ahh you're using the stable release of MemeBox which still uses the old 4444 port, the [nightly](https://github.com/negue/meme-box-nightly/releases) builds us the 6363 port

**Reply by negue** [reply]
Hey, that was a fast self fix ^^ 

The current release of meme-box (which is a bit old right now, yes we'll have a release soon, hopefully this month)  used 4444 - which conflicted with the OBS WebSocket Port (also 4444) 

Thats why we changed the default 6363 (nightly release) - I'd recommend using the nightly it has A-LOT of changes added and those will be hopefully soon release - I'm giving a lot of time to make it happen :)

**Reply by Glycode** [reply]
oh was I supposed to install the nightly build. my bad haha

**Reply by gigili** [reply]
I think it should be fine with using the stable version (@negue  can you confirm), but the nightly has more features in it :)

**Reply by Glycode** [reply]
Yes I'll install the nightly now, thank you :)

**Reply by negue** [reply]
Just had to test it and yes also works with the released version ^^ 

If any suggestions / feature requests / issues - let me know

**Reply by Glycode** [reply]
Thanks, appreciate the support ^^
Have the nightly now and playing a bit with everything, still relatively new to this :P
